tgui/packages/common/color.ts

```diff
@@ -61,15 +48,21 @@ export class Color {
     return this.darken(-percent);
   }
 
-  fromHex = (hex: string): Color => {
+  /**
+   * Creates a color from the CSS hex color notation.
+   */
+  static fromHex = (hex: string): Color => {
```

Comment:
Why not `static fromHex(hex: string): Color {`?

Reply:
(i hate javascript so much) what's the difference in the class? i did this so I could list `fromHex` as a property, otherwise it wasn't registering in typescript as existing.

i suppose now with static on `fromHex` directly it makes more sense to be on the actual method. 
is `varname = () => {` just an object on the class??

Reply:
- varname = smth is a property on the object
- static varname = smth is a property on the class
- varname() is a method accessible on the object
- static varname() is a method accessible on the class

Difference between arrow function and method is that on arrow function `this` is not rebindable, while `this` on a method call be rebound using `Class.method.call(newThis, ...args)`. In practice, not much difference, methods are just shorter to write. In React, using arrow functions on classes is useful for callbacks, because they must always have access to `this` of a Component.